runtime/autoload/remote/define.vim

```diff
@@ -169,14 +169,27 @@ function! remote#define#FunctionOnChannel(channel, method, sync, name, opts)
 endfunction
 
 
+function! remote#define#SafeRPCNofity(...) abort
+  if !exists('s:poll')
+    let s:poll = {}
+  endif
+  " check the process started correctly
+  if !get(s:poll, a:1, 0)
+    if rpcrequest(a:1, 'poll') ==# 'ok'
+      let s:poll[a:1] = 1
+    endif
+  endif
+  return call('rpcnotify', a:000)
```

Comment:
The check is to wait the provider initialization.
If the check is not exists, the initial rpcnotify will be error.